.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         uses: conda-incubator/setup-miniconda@v3
         with:
           python-version: "3.10"
-          mamba-version: "*"
+          mamba-version: "1"
```

Comment:
I had to restrict mamba to version 1 as I was running into the following issue: https://github.com/jiarong/VirSorter2/issues/222

Solution: https://github.com/jiarong/VirSorter2/issues/222#issuecomment-2452806948